public/app/plugins/datasource/tempo/resultTransformer.ts

```diff
@@ -488,7 +488,12 @@ export function transformTrace(response: DataQueryResponse, nodeGraph = false):
     return emptyDataQueryResponse;
   }
 
-  parseJsonFields(frame);
+  try {
+    parseJsonFields(frame);
+  } catch (error) {
+    console.error(error);
+    return { error: { message: 'JSON is not valid OpenTelemetry format: ' + error }, data: [] };
```

Comment:
Because the trace isn't being uploaded via JSON, we should update the error message to something like `Unable to parse trace response` or `Invalid trace data`